internal/controller/operatorconfigmap_controller.go

```diff
@@ -469,6 +481,45 @@ func (c *OperatorConfigMapReconciler) reconcileDelegatedCSI() error {
 	return nil
 }
 
+func (c *OperatorConfigMapReconciler) getDomainLabels() ([]string, error) {
+	topologyLabels := []string{
+		"kubernetes.io/hostname",
+		"topology.kubernetes.io/zone",
+		"topology.rook.io/rack",
+	}
+	actualTopologyLabels := []string{}
+
+	// list node labels
+	nodes := &metav1.PartialObjectMetadataList{}
+	nodes.SetGroupVersionKind(corev1.SchemeGroupVersion.WithKind("NodeList"))
+	labelSelector := &client.ListOptions{
+		LabelSelector: labels.SelectorFromSet(map[string]string{"node-role.kubernetes.io/worker": ""}),
```

Comment:
why we need label selector, what is there are cases where we need to run the csi daemonset on master node as we.. Having hardcoded checks will block it